code/game/area/areas/ruins/lavaland.dm

```diff
@@ -0,0 +1,88 @@
+//Lavaland Ruins
+
+/area/ruin
```

Comment:
TG has refactored this a bit, may want to look into that; it's mostly just pushing up vars to their highest point.